dir.targets

```diff
@@ -111,6 +121,29 @@
            Text="The build tools package was just restored and so we cannot continue the build of an individual project because targets from the build tools package were not able to be imported. Please retry the build the individual project again." />
   </Target>
 
+  <Target Name="DownloadCli">
+    <PropertyGroup>
+        <CliRootDir>$(ProjectDir)/bin/tools/cli/</CliRootDir>
+        <CliDownloadFileName Condition="'$(OSGroup)'=='Windows_NT'">dotnet-win-x64.latest.zip</CliDownloadFileName>
+        <CliDownloadFileName Condition="'$(OSGroup)'=='Linux'">dotnet-linux-x64.latest.tar.gz</CliDownloadFileName>
+        <CliDownloadFileName Condition="'$(OSGroup)'=='OSX'">dotnet-osx-x64.latest.tar.gz</CliDownloadFileName>
+        <CliDownloadFilePath>$(CliRootDir)$(CliDownloadFileName)</CliDownloadFilePath>
+        <CliFetchUrl>https://dotnetcli.blob.core.windows.net/dotnet/dev/Binaries/Latest/$(CliDownloadFileName)</CliFetchUrl>
+    </PropertyGroup>
+
+    <Message Text="Downloading CLI... from $(CliFetchUrl)" />
+
+    <DownloadFile FileName="$(CliDownloadFilePath)"
```

Comment:
How are we handling the failure in download (as opposed to build aborting later due to a missing file and cryptic error message)?